harness/determined/cli/master.py

```diff
@@ -29,16 +33,18 @@ def set_master_config(args: Namespace) -> None:
         log_config.level = bindings.v1LogLevel[args.log_level]
         field_masks.append("log.level")
 
+    if len(field_masks) == 0:
+        raise cli.errors.CliError(
+            "Please provide at least one argument to set master config. "
+            + "Currently, the supported fields are --log.level and --log.color."
+        )
```

Comment:
I'd suggest printing the general warning *after* this error.
otherwise, whenever this error will happen, users will be looking at the preceding warning and trying to figure out if it's the reason for the failure

Reply:
Makes sense, done